### PR TITLE
Cluster string in Placement is optional

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/Samples/TlsVariationsExample/TlsVariationsExample.csproj [hunk 1]
```diff
@@ -24,7 +24,7 @@
     </ItemGroup>
 
     <PropertyGroup>
-        <NoWarn>$(NoWarn);NU1701</NoWarn>
+        <NoWarn>$(NoWarn);NU1701;NU1902</NoWarn>
```
Comment (mtmk) [inline src/Samples/TlsVariationsExample/TlsVariationsExample.csproj:27]: was there a package marked as vulnerable?

Comment (scottf) [reply src/Samples/TlsVariationsExample/TlsVariationsExample.csproj:27]: Yes. In an example. Not in the actual client. Ignoring b/c it's irrelevant.

Comment (mtmk) [reply src/Samples/TlsVariationsExample/TlsVariationsExample.csproj:27]:
```
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-8xfc-gm6g-vgpv
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-m44j-cfrm-g8qc
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-v435-xc8x-wvr9
```

Comment (mtmk) [reply src/Samples/TlsVariationsExample/TlsVariationsExample.csproj:27]:
```
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-8xfc-gm6g-vgpv
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-m44j-cfrm-g8qc
src\Samples\TlsVariationsExample\TlsVariationsExample.csproj(0,0): Error NU1902: Warning As Error: Package 'BouncyCastle' 1.8.9 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-v435-xc8x-wvr9
```

Comment (mtmk) [reply src/Samples/TlsVariationsExample/TlsVariationsExample.csproj:27]: created #893 to fix later.

**Review by mtmk — APPROVED** [review]
LGTM

repo vulnerability warning should be fix later since it's in example code. (#893)

**Pull request merged**